src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,64 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { useSearchParams, Link } from 'react-router-dom';
+import cn from 'classnames';
+import { SexFilter } from '../types';
+import { getSearchWith } from '../utils/getSearchWith';
+
+type Props = {};
```

Comment:
You have declared an empty Props type here. If you do not expect any props for this component, you do not need to declare an empty Props type. You can directly use React.FC<{}> in your component declaration.